Tests/InsightsTests.swift

```diff
@@ -13,6 +13,7 @@ final class InsightsTests: XCTestCase {
     override func setUp() {
         super.setUp()
         EventManager.shared.deleteEvents(before: Date().epochMilliseconds())
+        Insights.clearInstance()
```

Comment:
Please remove this as since we added a separate test for clearInstance.

Reply:
Removed Insights.clearInstance()